specification/network/resource-manager/Microsoft.Network/stable/2019-06-01/virtualNetwork.json

```diff
@@ -1674,10 +1674,6 @@
           "type": "string",
           "description": "The name of the service for which subnet is being prepared for."
         },
-        "resourceGroupName": {
```

Comment:
Has this version shipped? Isn't this breaking?

Reply:
@ambhatt can you answer

Reply:
@Am018 Can you answer this?

Reply:
There was no work done (Powershell, CLI) on top of this implementation and this API is still behind feature flag and hence not getting used by anyone other than some internal MSFT teams for testing. Hence its a safe change.

Reply:
for feature flags we do strongly recommend to use private previews, not public one

Reply:
We are adding these changes in new api version (2019-06-01) folder isn't that good enough?

Reply:
@KrisBash  we are making these changes in new API version (2019-06-01). So it shouldn't be any issue. Please let me know if there is anything else I need to do for this PR.

Reply:
@MikhailTryakhov I had discussion with @Am018 and this API is available publicly but not used that's it. So we decided to make changes in new API version 2019-06-01 and publish changes in public repository only. So please take a look at the PR and approve it.